addons/sales_team/models/crm_team_member.py

```diff
@@ -13,7 +13,7 @@ class CrmTeamMember(models.Model):
     _check_company_auto = True
 
     crm_team_id = fields.Many2one(
-        'crm.team', string='Sales Team',
+        'crm.team', string='Sales Team', group_expand='_read_group_crm_team_ids',
```

Comment:
Would name it ``_read_group_crm_team_id`` (no "s", to match the field name :) )

Reply:
Hello @tde-banana-odoo 

Changes are done.

Thanks